src/Umbraco.Web/Editors/BackOfficeServerVariables.cs

```diff
@@ -52,7 +54,8 @@ public BackOfficeServerVariables(UrlHelper urlHelper, ApplicationContext applica
                 {"umbracoUrls", new[] {"authenticationApiBaseUrl", "serverVarsJs", "externalLoginsUrl", "currentUserApiBaseUrl"}},
                 {"umbracoSettings", new[] {"allowPasswordReset", "imageFileTypes", "maxFileSize", "loginBackgroundImage"}},
                 {"application", new[] {"applicationPath", "cacheBuster"}},
-                {"isDebuggingEnabled", new string[] { }}
+                {"isDebuggingEnabled", new string[] { }},
+                {"disabledFeatures", new [] {"disableTemplate"}}
```

Comment:
we should encapsulate this in "features" and then the child "disabledFeatures" making this future proof for if/when we added "enabledFeatures"